pkgs/development/python-modules/fpdf2/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, defusedxml
+, pillow
+, fonttools
+}:
+
+buildPythonPackage rec {
+  pname = "fpdf2";
+  version = "2.6.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-O7GfowpbHXb9FHCkdGTeXgqkzBynKWE/3RB2gXrv/kM=";
+  };
+
+  propagatedBuildInputs = [ defusedxml pillow fonttools ];
+
+  doCheck = false;
+
+  pythonImportsCheck = [ "fpdf" ];
+
+  meta = with lib; {
```

Comment:
Please order the meta fields as outlined [here](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md)